Source/Controllers/MainViewControllers/SPDatabaseDocument.m

```diff
@@ -5385,7 +5391,7 @@ - (void)_selectDatabaseAndItem:(NSDictionary *)selectionDetails
             [[chooseDatabaseButton onMainThread] selectItemWithTitle:targetDatabaseName];
 
             selectedDatabase = [[NSString alloc] initWithString:targetDatabaseName];
-            selectedTableName = targetItemName ? [[NSString alloc] initWithString:targetItemName] : nil;
+            selectedTableName = nil;
```

Comment:
Why remove the targetItemName line here? Won't this break some selection of tables?